src/average.js

```diff
@@ -12,6 +12,28 @@
     - average([1, '2']) // Retorno: undefined;
 */
 
-const average = () => {};
+function verificaArr(arr) {
+  if (arr === null || arr.length === 0) {
+    return true;
+  }
+  for (const num in arr) {
+    if (typeof arr[num] !== 'number') {
+      return true;
+    }
+  }
+  return false;
+}
+
+const average = (arr) => {
```

Comment:
O que acha de dar um nome mais descritivo ao seu parâmetro?
```suggestion
const average = (numbers) => {
```